### PR TITLE
Cut release v0.4.0

### LINKED ISSUE #86: Cut a new release
Since `v0.3.0` a couple of nice additions have landed (#65, #67 and #70). Maybe its tome for a new release?

### PR DESCRIPTION
**Pull request by jan--f**
This also pulls in #79.

Fixes #86

### CODE REVIEW

**Comment by jan--f** [comment]
Supersedes #87

**Comment by jan--f** [comment]
cc @squat @simonpasquier

**Review by squat — APPROVED** [review]
LGTM
Thanks @jan--f

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.4.0 / 2021-09-29
```
Comment (simonpasquier) [inline CHANGELOG.md:1]: just one nit
```suggestion
## 0.4.0 / 2021-10-05
```

**Review by simonpasquier — APPROVED** [review]
lgtm

**Pull request merged**